tests/utils/getElementsUtils.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+export const getByDataCy = (dataCy, parent = document) => {
+  return parent.querySelector(`[data-cy="${dataCy}"]`);
+};
+
+export const getAllByDataCyRegEx = (dataCyRegEx, parent = document) => {
+  const output = [];
+  for (const i of parent.querySelectorAll('*')) {
```

Comment:
yes thanks. It's a old compatiilty trick here for IE ;) because querySelectorAll return a NodeList, not array and the forEach() method wasn't supported by old browser.
Now it's fine ;)